Modules/Filtering/ImageIntensity/test/itkVectorToRGBImageAdaptorTest.cxx

```diff
@@ -81,12 +81,7 @@ itkVectorToRGBImageAdaptorTest(int, char *[])
   vector[2] = 1.4;
 
   // Initializing all the pixel in the image
-  it1.GoToBegin();
-  while (!it1.IsAtEnd())
-  {
-    it1.Set(vector);
-    ++it1;
-  }
+  image->FillBuffer(vector);
```

Comment:
Note that there is another `while` loop a little bit further down here, that I did not touch with this pull request: https://github.com/InsightSoftwareConsortium/ITK/blob/07f5450a715c3f17e4fe0f140724f59cd02e5432/Modules/Filtering/ImageIntensity/test/itkVectorToRGBImageAdaptorTest.cxx#L107-L124 

I _think_ the original intention was to test that `RGBIteratorType it2(adaptor, adaptor->GetRequestedRegion())` works well with an adaptor, so then, replacing this with a `FillBuffer` call might not be according to the original intention 🤷